core/src/main/java/brooklyn/entity/rebind/dto/MementosGenerators.java

```diff
@@ -68,10 +71,23 @@
 
     private MementosGenerators() {}
     
+    /** @deprecated since 0.7.0 use {@link #newBasicMemento(BrooklynObject)} */
```

Comment:
This isn't the real memento as per new java doc.  The old name confused me
and caused the manual export failures.
On 18 Mar 2015 11:59, "Aled Sage" <notifications@github.com> wrote:

> In core/src/main/java/brooklyn/entity/rebind/dto/MementosGenerators.java
> <https://github.com/apache/incubator-brooklyn/pull/554#discussion_r26656722>
> :
>
> > @@ -68,10 +71,23 @@
> >
> >      private MementosGenerators() {}
> >
> > +    /** @deprecated since 0.7.0 use {@link #newBasicMemento(BrooklynObject)} */
>
> Why rename from newMemento(BrooklynObject) to
> newBasicMemento(BrooklynObject)? Do we have anything but "basic", or are
> we expecting that in the near future? Personally I prefer newMemento, as
> "basic" implies there is some other kind of memento out there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-brooklyn/pull/554/files#r26656722>.
>